worker/src/queues/ses/helpers/FileService.ts

```diff
@@ -16,6 +29,10 @@ export default class FileService {
    */
   async getFile(url: string): Promise<{ contentType: string; data: Buffer }> {
     try {
+      const isValid = this.validateFileLocation(url);
```

Comment:
move this outside of the try block so you don't need to throw, catch and rethrow